src/types/BannerTypes.ts

```diff
@@ -92,3 +94,28 @@ export interface BannerProps {
     tickerSettings?: TickerSettings;
     submitComponentEvent?: (componentEvent: OphanComponentEvent) => void;
 }
+
+export interface ArticlesViewedSettings {
+    minViews: number;
+    maxViews: number;
+    periodInWeeks: number;
+}
+
+export interface RawVariantParams {
+    name: string;
+    body: string;
+    highlightedText?: string;
+    cta?: Cta;
+    secondaryCta?: Cta;
+}
+
+export interface RawTestParams {
+    name: string;
+    nickname: string;
+    isOn: boolean;
+    minArticlesBeforeShowingBanner: number;
+    userCohort: BannerAudience;
+    locations: CountryGroupId[];
+    variants: RawVariantParams[];
+    articlesViewedSettings: ArticlesViewedSettings;
```

Comment:
I've added a card for this field, again for a follow up PR

Reply:
👍